src/Cdn/Cdn/ChangeLog.md

```diff
@@ -17,10 +17,12 @@
     * Overview of change #1
         - Additional information about change #1
 -->
-## Upcoming Release
 
 ## Version 1.0.1
 * Update incorrect online help URLs
 
 ## Version 1.0.0
 * General availability of `Az.Cdn` module
+
+## Upcoming Release
```

Comment:
Please move this back to the top where it was originally - we depend on the structure of file when doing automation around release versioning.

Reply:
Done